2019-11-07 13:51:53 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:53 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:53 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
2019-11-07 13:51:53 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:53 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:53 default[20191107t191555]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2019-11-07 13:51:53 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2019-11-07 13:51:53 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2019-11-07 13:51:53 default[20191107t191555]  	at java.base/java.lang.Thread.run(Thread.java:834)
2019-11-07 13:51:53 default[20191107t191555]  ============= end nested exception, level (2) ===========
2019-11-07 13:51:53 default[20191107t191555]  Cleanup action completed
2019-11-07 13:51:54 default[20191107t191555]  2019-11-07 13:51:54.360 ERROR 16 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.
2019-11-07 13:51:54 default[20191107t191555]
2019-11-07 13:51:54 default[20191107t191555]  java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]  	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]  Caused by: org.apache.derby.iapi.error.StandardException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 15 common frames omitted  Caused by: org.apache.derby.iapi.error.StandardException: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 12 common frames omitted
2019-11-07 13:51:54 default[20191107t191555]
2019-11-07 13:51:54 default[20191107t191555]  2019-11-07 13:51:54.362  WARN 16 --- [           main] o.s.b.a.orm.jpa.DatabaseLookup           : Unable to determine jdbc url from datasource
2019-11-07 13:51:54 default[20191107t191555]
2019-11-07 13:51:54 default[20191107t191555]  org.springframework.jdbc.support.MetaDataAccessException: Could not get Connection for extracting meta data; nested exception is org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:327) ~[spring-jdbc-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:355) ~[spring-jdbc-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.DatabaseLookup.getDatabase(DatabaseLookup.java:72) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.JpaProperties.determineDatabase(JpaProperties.java:168) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.JpaBaseConfiguration.jpaVendorAdapter(JpaBaseConfiguration.java:111) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaConfiguration$$EnhancerBySpringCGLIB$$7383b0bb.CGLIB$jpaVendorAdapter$3(<generated>) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaConfiguration$$EnhancerBySpringCGLIB$$7383b0bb$$FastClassBySpringCGLIB$$bc28e5d4.invoke(<generated>) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:228) ~[spring-core-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:361) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaConfiguration$$EnhancerBySpringCGLIB$$7383b0bb.jpaVendorAdapter(<generated>) ~[spring-boot-autoconfigure-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]  	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]  	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:579) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1250) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:502) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:312) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:310) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:251) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1138) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1065) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:815) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:721) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:470) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1250) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:502) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:312) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:310) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:251) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1138) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1065) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:815) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:721) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:470) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1250) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:545) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:502) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:312) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:310) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1085) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:858) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:140) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:388) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:327) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1246) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1234) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at com.in28minutes.springboot.tutorial.basics.application.configuration.SpringBootWebApplicationBootstrapJqueryApplication.main(SpringBootWebApplicationBootstrapJqueryApplication.java:10) ~[classes!/:0.0.1-SNAPSHOT]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]  	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]  	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  Caused by: org.springframework.jdbc.CannotGetJdbcConnectionException: Failed to obtain JDBC Connection; nested exception is java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.springframework.jdbc.datasource.DataSourceUtils.getConnection(DataSourceUtils.java:81) ~[spring-jdbc-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.jdbc.support.JdbcUtils.extractDatabaseMetaData(JdbcUtils.java:318) ~[spring-jdbc-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	... 69 common frames omitted  Caused by: java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]  	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]  Caused by: org.apache.derby.iapi.error.StandardException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 15 common frames omitted  Caused by: org.apache.derby.iapi.error.StandardException: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 12 common frames omitted
2019-11-07 13:51:54 default[20191107t191555]
2019-11-07 13:51:54 default[20191107t191555]  2019-11-07 13:51:54.396  INFO 16 --- [           main] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2019-11-07 13:51:54 default[20191107t191555]  2019-11-07 13:51:54.396  WARN 16 --- [           main] com.zaxxer.hikari.util.DriverDataSource  : Registered driver with driverClassName=org.apache.derby.jdbc.EmbeddedDriver was not found, trying direct instantiation.
2019-11-07 13:51:54 default[20191107t191555]  Thu Nov 07 13:51:54 UTC 2019 Thread[Thread-4,5,main] Cleanup action starting
2019-11-07 13:51:54 default[20191107t191555]  ERROR XBM0H: Directory /srv/mydb cannot be created.
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.lang.Thread.run(Thread.java:834)
2019-11-07 13:51:54 default[20191107t191555]  Cleanup action completed
2019-11-07 13:51:54 default[20191107t191555]  Thu Nov 07 13:51:54 UTC 2019 Thread[Thread-4,5,main] Cleanup action starting
2019-11-07 13:51:54 default[20191107t191555]  java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)  	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)  	at java.base/java.lang.Thread.run(Thread.java:834)  Caused by: ERROR XJ041: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)  	... 15 more  Caused by: ERROR XBM0H: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source)  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source)  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source)  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source)  	... 12 more
2019-11-07 13:51:54 default[20191107t191555]  ============= begin nested exception, level (1) ===========
2019-11-07 13:51:54 default[20191107t191555]  java.sql.SQLException: Directory /srv/mydb cannot be created.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)  	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)  	at org.apache.derby.impl.jdbc.TransactionResourceImpl.wrapInSQLException(Unknown Source)  	at org.apache.derby.impl.jdbc.TransactionResourceImpl.handleException(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection.handleException(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)  	at java.base/java.lang.Thread.run(Thread.java:834)  Caused by: ERROR XBM0H: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source)  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source)  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source)  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source)  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source)  	at java.base/java.security.AccessController.doPrivileged(Native Method)  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source)  	... 12 more
2019-11-07 13:51:54 default[20191107t191555]  ============= end nested exception, level (1) ===========
2019-11-07 13:51:54 default[20191107t191555]  ============= begin nested exception, level (2) ===========
2019-11-07 13:51:54 default[20191107t191555]  ERROR XBM0H: Directory /srv/mydb cannot be created.
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.lang.Thread.run(Thread.java:834)
2019-11-07 13:51:54 default[20191107t191555]  ============= end nested exception, level (2) ===========
2019-11-07 13:51:54 default[20191107t191555]  Cleanup action completed
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.402 ERROR 16 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]  	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]  Caused by: org.apache.derby.iapi.error.StandardException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 15 common frames omitted  Caused by: org.apache.derby.iapi.error.StandardException: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 12 common frames omitted
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.451  INFO 16 --- [           main] j.LocalContainerEntityManagerFactoryBean : Building JPA container EntityManagerFactory for persistence unit 'default'
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.488  INFO 16 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
2019-11-07 13:51:55 default[20191107t191555]  	name: default
2019-11-07 13:51:55 default[20191107t191555]  	...]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.710  INFO 16 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.2.14.Final}
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.712  INFO 16 --- [           main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.753  WARN 16 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.757  INFO 16 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.782  INFO 16 --- [           main] ConditionEvaluationReportLoggingListener :
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.786 ERROR 16 --- [           main] o.s.boot.SpringApplication               : Application run failed
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1710) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:583) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:502) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:312) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:310) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1085) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:858) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:140) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:388) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:327) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1246) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1234) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at com.in28minutes.springboot.tutorial.basics.application.configuration.SpringBootWebApplicationBootstrapJqueryApplication.main(SpringBootWebApplicationBootstrapJqueryApplication.java:10) ~[classes!/:0.0.1-SNAPSHOT]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]  	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]  	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException  	at org.hibernate.boot.spi.XmlMappingBinderAccess.<init>(XmlMappingBinderAccess.java:43) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.boot.MetadataSources.<init>(MetadataSources.java:87) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.<init>(EntityManagerFactoryBuilderImpl.java:209) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.<init>(EntityManagerFactoryBuilderImpl.java:164) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.springframework.orm.jpa.vendor.SpringHibernateJpaPersistenceProvider.createContainerEntityManagerFactory(SpringHibernateJpaPersistenceProvider.java:51) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.createNativeEntityManagerFactory(LocalContainerEntityManagerFactoryBean.java:365) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.AbstractE2019-11-07 13:51:54 default[20191107t191555]  ============= begin nested exception, level (2) ===========
2019-11-07 13:51:54 default[20191107t191555]  ERROR XBM0H: Directory /srv/mydb cannot be created.
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.security.AccessController.doPrivileged(Native Method)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2019-11-07 13:51:54 default[20191107t191555]  	at java.base/java.lang.Thread.run(Thread.java:834)
2019-11-07 13:51:54 default[20191107t191555]  ============= end nested exception, level (2) ===========
2019-11-07 13:51:54 default[20191107t191555]  Cleanup action completed
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.402 ERROR 16 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  java.sql.SQLException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createDatabase(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]  	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]  Caused by: org.apache.derby.iapi.error.StandardException: Failed to create database 'mydb', see the next exception for details.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 15 common frames omitted  Caused by: org.apache.derby.iapi.error.StandardException: Directory /srv/mydb cannot be created.  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService$10.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.services.monitor.StorageFactoryService.createServiceRoot(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.BaseMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.services.monitor.FileMonitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.iapi.services.monitor.Monitor.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at org.apache.derby.impl.jdbc.EmbedConnection$5.run(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	at java.base/java.security.AccessController.doPrivileged(Native Method) ~[na:na]  	at org.apache.derby.impl.jdbc.EmbedConnection.createPersistentService(Unknown Source) ~[derby-10.14.1.0.jar!/:na]  	... 12 common frames omitted
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.451  INFO 16 --- [           main] j.LocalContainerEntityManagerFactoryBean : Building JPA container EntityManagerFactory for persistence unit 'default'
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.488  INFO 16 --- [           main] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [
2019-11-07 13:51:55 default[20191107t191555]  	name: default
2019-11-07 13:51:55 default[20191107t191555]  	...]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.710  INFO 16 --- [           main] org.hibernate.Version                    : HHH000412: Hibernate Core {5.2.14.Final}
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.712  INFO 16 --- [           main] org.hibernate.cfg.Environment            : HHH000206: hibernate.properties not found
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.753  WARN 16 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.757  INFO 16 --- [           main] o.apache.catalina.core.StandardService   : Stopping service [Tomcat]
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.782  INFO 16 --- [           main] ConditionEvaluationReportLoggingListener :
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2019-11-07 13:51:55 default[20191107t191555]  2019-11-07 13:51:55.786 ERROR 16 --- [           main] o.s.boot.SpringApplication               : Application run failed
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 13:51:55 default[20191107t191555]  org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1710) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:583) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:502) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:312) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:228) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:310) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:200) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1085) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:858) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:549) ~[spring-context-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:140) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:752) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:388) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:327) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1246) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1234) ~[spring-boot-2.0.0.RELEASE.jar!/:2.0.0.RELEASE]  	at com.in28minutes.springboot.tutorial.basics.application.configuration.SpringBootWebApplicationBootstrapJqueryApplication.main(SpringBootWebApplicationBootstrapJqueryApplication.java:10) ~[classes!/:0.0.1-SNAPSHOT]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:na]  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]  	at java.base/java.lang.reflect.Method.invoke(Method.java:566) ~[na:na]  	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  Caused by: java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException  	at org.hibernate.boot.spi.XmlMappingBinderAccess.<init>(XmlMappingBinderAccess.java:43) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.boot.MetadataSources.<init>(MetadataSources.java:87) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.<init>(EntityManagerFactoryBuilderImpl.java:209) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.<init>(EntityManagerFactoryBuilderImpl.java:164) ~[hibernate-core-5.2.14.Final.jar!/:5.2.14.Final]  	at org.springframework.orm.jpa.vendor.SpringHibernateJpaPersistenceProvider.createContainerEntityManagerFactory(SpringHibernateJpaPersistenceProvider.java:51) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.createNativeEntityManagerFactory(LocalContainerEntityManagerFactoryBean.java:365) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.buildNativeEntityManagerFactory(AbstractEntityManagerFactoryBean.java:388) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.AbstractEntityManagerFactoryBean.afterPropertiesSet(AbstractEntityManagerFactoryBean.java:377) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean.afterPropertiesSet(LocalContainerEntityManagerFactoryBean.java:341) ~[spring-orm-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1769) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1706) ~[spring-beans-5.0.4.RELEASE.jar!/:5.0.4.RELEASE]  	... 24 common frames omitted  Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException  	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[na:na]  	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[na:na]  	at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:93) ~[spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar:0.0.1-SNAPSHOT]  	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[na:na]  	... 35 common frames omitted
2019-11-07 13:51:55 default[20191107t191555]
2019-11-07 14:10:44 default[20191107t193800]  "GET / HTTP/1.1" 404
2019-11-07 14:10:47 default[20191107t193800]
2019-11-07 14:10:47 default[20191107t193800]    .   ____          _            __ _ _
2019-11-07 14:10:47 default[20191107t193800]   /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
2019-11-07 14:10:47 default[20191107t193800]  ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
2019-11-07 14:10:47 default[20191107t193800]   \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
2019-11-07 14:10:47 default[20191107t193800]    '  |____| .__|_| |_|_| |_\__, | / / / /
2019-11-07 14:10:47 default[20191107t193800]   =========|_|==============|___/=/_/_/_/
2019-11-07 14:10:47 default[20191107t193800]   :: Spring Boot ::        (v2.0.0.RELEASE)
2019-11-07 14:10:47 default[20191107t193800]
2019-11-07 14:10:47 default[20191107t193800]  2019-11-07 14:10:47.890  INFO 15 --- [           main] WebApplicationBootstrapJqueryApplication : Starting SpringBootWebApplicationBootstrapJqueryApplication v0.0.1-SNAPSHOT on localhost with PID 15 (/srv/spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar started by www-data in /srv)
2019-11-07 14:10:47 default[20191107t193800]  2019-11-07 14:10:47.897  INFO 15 --- [           main] WebApplicationBootstrapJqueryApplication : No active profile set, falling back to default profiles: default
2019-11-07 14:10:48 default[20191107t193800]  2019-11-07 14:10:48.085  INFO 15 --- [           main] ConfigServletWebServerApplicationContext : Refreshing org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@1f7030a6: startup date [Thu Nov 07 14:10:48 UTC 2019]; root of context hierarchy
2019-11-07 14:10:49 default[20191107t193800]  WARNING: An illegal reflective access operation has occurred
2019-11-07 14:10:49 default[20191107t193800]  WARNING: Illegal reflective access by org.springframework.cglib.core.ReflectUtils$1 (jar:file:/srv/spring-boot-web-application-bootstrap-jquery-0.0.1-SNAPSHOT.jar!/BOOT-INF/lib/spring-core-5.0.4.RELEASE.jar!/) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
2019-11-07 14:10:49 default[20191107t193800]  WARNING: Please consider reporting this to the maintainers of org.springframework.cglib.core.ReflectUtils$1
2019-11-07 14:10:49 default[20191107t193800]  WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
2019-11-07 14:10:49 default[20191107t193800]  WARNING: All illegal access operations will be denied in a future release
2019-11-07 14:10:50 default[20191107t193800]  2019-11-07 14:10:50.932  INFO 15 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2019-11-07 14:10:51 default[20191107t193800]  2019-11-07 14:10:51.012  INFO 15 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2019-11-07 14:10:51 default[20191107t193800]  2019-11-07 14:10:51.012  INFO 15 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet Engine: Apache Tomcat/8.5.28
2019-11-07 14:10:51 default[20191107t193800]  2019-11-07 14:10:51.063  INFO 15 --- [ost-startStop-1] o.a.catalina.core.AprLifecycleListener   : The APR based Apache Tomcat Native library which allows optimal performance in production environments was not found on the java.library.path: [/usr/java/packages/lib:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib]
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.345  INFO 15 --- [ost-startStop-1] org.apache.jasper.servlet.TldScanner     : At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time.
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.681  INFO 15 --- [ost-startStop-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.682  INFO 15 --- [ost-startStop-1] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 4609 ms
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.904  INFO 15 --- [ost-startStop-1] o.s.b.w.servlet.ServletRegistrationBean  : Servlet dispatcherServlet mapped to [/]
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.911  INFO 15 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'characterEncodingFilter' to: [/*]
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.911  INFO 15 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'hiddenHttpMethodFilter' to: [/*]
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.911  INFO 15 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'httpPutFormContentFilter' to: [/*]
2019-11-07 14:10:52 default[20191107t193800]  2019-11-07 14:10:52.912  INFO 15 --- [ost-startStop-1] o.s.b.w.servlet.FilterRegistrationBean   : Mapping filter: 'requestContextFilter' to: [/*]
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.639  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerAdapter : Looking for @ControllerAdvice: org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext@1f7030a6: startup date [Thu Nov 07 14:10:48 UTC 2019]; root of context hierarchy
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.844  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/welcome]}" onto public java.lang.String com.in28minutes.springboot.tutorial.basics.application.configuration.WelcomeController.loginMessage()
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.856  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/login]}" onto public java.lang.String com.in28minutes.springboot.tutorial.basics.application.configuration.loginManager.loginController.loginPage()
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.857  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/createUser]}" onto public java.lang.String com.in28minutes.springboot.tutorial.basics.application.configuration.loginManager.loginController.createUserPage()
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.857  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/addNewUser],methods=[POST]}" onto public void com.in28minutes.springboot.tutorial.basics.application.configuration.loginManager.loginController.addnewUser(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse)
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.858  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/dologin],methods=[POST]}" onto public void com.in28minutes.springboot.tutorial.basics.application.configuration.loginManager.loginController.doLogin(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse)
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.859  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/]}" onto public java.lang.String com.in28minutes.springboot.tutorial.basics.application.configuration.homePageManager.homePageController.rootPage()
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.860  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/home]}" onto public java.lang.String com.in28minutes.springboot.tutorial.basics.application.configuration.homePageManager.homePageController.createUserPage()
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.860  INFO 15 --- [           main] s.w.s.m.m.a.RequestMappingHandlerMapping : Mapped "{[/searchMovie],methods=[POST]}" onto public void com.in28minutes.springboot.tutorial.basics.application.configuration.homePageManager.homePageController.searchMovie(javax.servlet.http.HttpServletRequest,javax.servlet.http.HttpServletResponse)
2019-11-07 14:10:53 default[20191107t193800]  2019-11-07 14:10:53.868  INFO 15 --- [